modules/yieldmoBidAdapter.js

```diff
@@ -370,7 +369,7 @@ function getPageDescription() {
  * @returns an id if there is one, or undefined
  */
 function getId(request, idType) {
-  return (typeof deepAccess(request, 'userId') === 'object') ? request.userId[idType] : undefined;
+  return (typeof deepAccess(request, 'userIdAsEids') === 'object') ? request.userIdAsEids[idType] : undefined;
```

Comment:
what was your reason for modifying this? as it is, it cannot be correct:

`request.userId` is a map; this method (before your change) returns one item from that map;
`request.userIdAsEids` is an array; this method now attempts to treat it as a map to look for an item in it. After your change, this method will *always* return undefined.